docs/management/managing-saved-objects.asciidoc

```diff
@@ -37,12 +37,15 @@ and select *Relationships*.
 [[managing-saved-objects-export-objects]]
 === Import and export
 
-Using the import and export commands, you can move objects between different
+Using the import and export actions, you can move objects between different
 {kib} instances. This action is useful when you
 have multiple environments for development and production.
 Import and export also work well when you have a large number
 of objects to update and want to batch the process.
 
+In addition to the user interface, {kib} provides beta <<saved-objects-api-import, import>> and <<saved-objects-api-export, export>> APIs if
+you want to automate this process.
+
```

Comment:
Added this here because I removed it from the Spaces docs (`docs/spaces/index.asciidoc`) but I still think it could be valuable.

We could also move this further down and/or put it in a `TIP` admonition. Thoughts?

Reply:
It looks good where you put it.